src/main/resources/templates/db_scripts/schema.json

```diff
@@ -249,6 +249,25 @@
           "tOps": "ADD"
         }
       ]
+    },
+    {
+      "tableName": "ledgerFY",
+      "fromModuleVersion": 4.0,
+      "withMetadata": true,
```

Comment:
`ledgerFY.json` does not have `metadata` so this can be set to `false` to avoid extra triggers which are not going to work anyway...

Reply:
done